### PR TITLE
Update package.json of Chart.js

### PR DESCRIPTION
**Pull request by simonbrunel**
Updating the Chart.js `package.json` to now npm auto update. Starting [this commit](https://github.com/chartjs/Chart.js/commit/9082507c1ccdbe1975519f49071d7e441de2a56d), dist files have been removed from the repository. See https://github.com/chartjs/Chart.js/pull/2555 for details.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@simonbrunel Please fix the indent, thanks.

**Comment by simonbrunel** [comment]
I guess that's because I inlined the `[{` `}]` for the `npmFileMap` property, so hopefully should be correct now. Can you please squash on merge? I did these changes directly in GitHub, cloning the repo takes forever.

File: ajax/libs/Chart.js/package.json [hunk 1]
```diff
@@ -8,20 +8,18 @@
...
-      "Chart.min.js",
-      "Chart.js"
-    ]
-  },
+  "npmName": "chart.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/Chart.js/package.json]: Let's use `"**/*"` here.

**Comment by PeterDaveHello** [comment]
Yes I can squash them.

**Comment by simonbrunel** [comment]
Thanks @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@simonbrunel no probs

**Pull request merged**